src/styles/answersStyles.js

```diff
@@ -23,7 +24,18 @@ border: none;
 
 //Styles for Answer.js
 
-export const AnswerSection = styled.div `
+export const AnswerContainer = styled.div`
```

Comment:
I styled our Answer, AnswerList, and some styles on QuestionList. Check them out and let me know if you want to keep mine, incorporate yours, or what. Thanks! Not sure why this wouldn't just let me merge everything anyway.